src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -313,7 +335,6 @@ const styles = {
         display: "inline-block",
```

Comment:
In my tests, I had to remove this `display: "inline-block"`, this way the whole components uses all the width it has, which is the most versatile thing to do, the parent div can decide then its width.

Reply:
Okay, I will try to remove it then and see how it looks like on metadata-synchronization too.